lib/openscap_parser/group.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+module OpenscapParser
+  class Group < XmlNode
+    def id
+      @id ||= parsed_xml['id']
+    end
+
+    def title
+      @title ||= parsed_xml.at_css('title') &&
+        parsed_xml.at_css('title').text
+    end
+
+    def description
+      @description ||= @parsed_xml.at_css('description') &&
+        @parsed_xml.at_css('description').text
+    end
+
+    def requires
+      @requires ||= parsed_xml.at_css('requires') &&
+        parsed_xml.at_css('requires')["idref"].split
+    end
+
+    def conflicts
+      @conflicts ||= parsed_xml.at_css('conflicts') &&
+        parsed_xml.at_css('conflicts')["idref"]
+    end
```

Comment:
@skateman I also added the `requires` and `conflicts` method to Group and Rule. The specification for XCCDF said that `requires` will be an element with an `idref` attribute that could look like `idref="A B C"` meaning either A, B or C is required so that's why I added `.split` onto the end in that method

Reply:
Are we sure that the conflicts can only have one item?

Reply:
<img width="613" alt="Screen Shot 2022-01-26 at 8 22 33 AM" src="https://user-images.githubusercontent.com/87209745/151180324-4e47298a-7a67-4013-b004-e84a1469fbc4.png">

Yeah, here's what it says in the xccdf specifications.. This is the part that makes it sound like it will only have one item in idref: 
`Each <xccdf:conflicts> element specifies a single conflicting item using its @idref attribute.`

Reply:
The 0-n column is the count column and it means that <conflicts> can show up more than once under the parent element

Reply:
I'd try this on real datastreams, just to be sure...

Reply:
Yeah, good idea! I tested with conflicts and requires that I added in a test xml myself but I can try to find one that actually has real requires or conflicts